projects/portal/src/app/pages/dialogs/delegate/delegate-form-dialog/delegate-form-dialog.component.ts

```diff
@@ -31,8 +34,14 @@ export class DelegateFormDialogComponent implements OnInit {
     private readonly walletService: WalletService,
     private readonly configS: ConfigService,
     private readonly stakingAppService: StakingApplicationService,
+    private readonly snackBar: MatSnackBar,
+    private readonly dialog: MatDialog,
   ) {
     this.validator = data;
+    this.validatorsList$ = this.cosmosSDK.sdk$.pipe(
+      mergeMap((sdk) => rest.staking.validators(sdk.rest)),
+      map((result) => result.data.validators),
+    );
```

Comment:
@Senna46
Sorry, I missed that.

I think this should be executed in pages component and pass it to this component.

Reply:
This is pages component, isn't it?

Reply:
Sorry it's just my mistake.
Now you remove unnecessary subscribe from this component, it's OK.